### ISSUE TITLE
 Show reason to approver why share is requested by the consumer and vice versa

### ISSUE THREAD

**Issue by manjulaK** [awslabs/aws-dataall#422]
### Describe the bug

Currently when the consumer requests a data sets he provides a reason for requesting the data set. however this field does not populate for the approver. Similarly when the approver rejects a request the reason for rejection is not shown to the consumer. UI needs to change and this is critical in a governance workflow.

### How to Reproduce

```
*P.S. Please do not attach files as it's considered a security risk. Add code snippets directly in the message body as much as possible.*
```

### Expected behavior

The approver of the request should also see the reason filled by the consumer for requesting the dataset in data.all UI. Similarly the consumer needs to see the reason for revoke or rejection of his request.

### OS

All

### Python version

3.1

### AWS data.all version

v1.3,v1.4

**Comment by dlpzx** [comment]
Thank you for opening an issue. At the moment we are focused on code-refractoring, so we won't be implementing it right away. But we are happy to support you in the implementation

**Comment by manjulaK** [comment]
hi @dlpzx  this is a must have for some customers and truly is a bug today. Please consider fixing for v1.6

**Comment by noah-paige** [comment]
Implemented as part of v1.6
